lib/common

```diff
@@ -208,3 +208,10 @@ get_allocated_nodes() {
 
   get_allocated_nodes_retval=$allocated_nodes
 }
+
+# each pacemaker cluster should have one founder node
+get_cluster_founders() {
+
+  local founders=$(knife search node "pacemaker_founder:true AND pacemaker_config_environment:*" -a name | grep ^name: | cut -d : -f 2)
```

Comment:
Heh. My comment was specifically saying that this will work only if the barclamp is not deactivated, but we do that in the `pre` script. The solution for this is to actually fetch the cluster founders before deactivating the barclamps, and save them in a file. And then parse this file when we need the list of founders.